src/app.rs

```diff
@@ -1462,6 +1463,33 @@ This flag can be disabled with --no-search-zip.
     args.push(arg);
 }
 
+fn flag_preprocessor(args: &mut Vec<RGArg>) {
+    const SHORT: &str = "search outputs of \"COMMAND FILE\" for each FILE";
```

Comment:
Quoting `COMMAND FILE` doesn't seem right; place-holders aren't quoted anywhere else in the help text. Also you switch between `COMMAND FILE` and `COMMAND`; `COMMAND` seems simpler (or maybe `PROGRAM` if you're worried that's too vague). Also `deactivates` is misspelt

Reply:
Ok.  Dequoted & fixed spelling.  I use COMMAND FILE when there is also a FILE in the sentence context and COMMAND when it's just about the program.

Reply:
Ah, i think i see what you mean